provider/common/bootstrap.go

```diff
@@ -156,7 +157,7 @@ func BootstrapInstance(
 			if icfg.AgentEnvironment == nil {
 				icfg.AgentEnvironment = make(map[string]string)
 			}
-			icfg.AgentEnvironment[agent.LxcBridge] = args.ContainerBridgeName
+			icfg.AgentEnvironment[agent.LxdBridge] = args.ContainerBridgeName
```

Comment:
Ah... ok. Is this actually used anywhere?

Reply:
Used in lxdBroker StartInstance()

```
	// Something to fallback to if there are no devices given in args.NetworkInfo
	// TODO(jam): 2017-02-07, this feels like something that should never need
	// to be invoked, because either StartInstance or
	// prepareOrGetContainerInterfaceInfo should always return a value. The
	// test suite currently doesn't think so, and I'm hesitant to munge it too
	// much.
	bridgeDevice := broker.agentConfig.Value(agent.LxdBridge)
	if bridgeDevice == "" {
		bridgeDevice = broker.agentConfig.Value(agent.LxcBridge)
	}
	if bridgeDevice == "" {
		bridgeDevice = network.DefaultLXDBridge
	}
```